website/src/plugins/remark-npm2yarn.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// This is a very naive implementation of converting npm commands to yarn commands
+// Works well for our use case since we only use either 'npm install', or 'npm run <something>'
+// Its impossible to convert it right since some commands at npm are not available in yarn and vice/versa
+const convertNpmToYarn = npmCode => {
+  // global install: 'npm i' -> 'yarn'
+  return (
+    npmCode
+      .replace(/^npm i$/gm, 'yarn')
+      // install: 'npm install --save foo' -> 'yarn add foo'
+      .replace(/npm install --save/gm, 'yarn add')
```

Comment:
Wanna handle the `npm install --save-dev` scenario too? Although we won't likely need it.

Reply:
most likely we wont have dev i guess. Its hard to find perfect solution to convert npm 2 yarn